### ISSUE TITLE
about elp-scatterplot-P0-Pk.html 

### ISSUE THREAD

**Issue by pierreganty** [elenagutiv/Linearisation-2015#11]
I should have the following. 
Pick an example and let y0 be its runtime for P0.
Then on the horizontal line y=y0, I should find the min(P1,P2) runtimes for the values of k=1,2,3,4,5 provided none of them times out.

**Comment by elenagutiv** [comment]
It has sense..So one point per P0 (instead of 5 points), right?

**Comment by pierreganty** [comment]
Yes. Each example should have the following points:
(rtP0, rtPmin_k=1)
...
 (rtP0, rtPmin_k=5)

where rtPX means runtime for PX.
Note that in the left column the value rtP0 is always the same.

You comment confuses me even more, what do you have right now?
Do you run QARMC on P0 five times and have possibly different values in the left column?

**Comment by elenagutiv** [comment]
Definitely, that is what im doing... im afraid i added a new line in the
script to generate P0 time but in the wrong place. I will change that and
generate the graph again.

Basically, what i have now is
(rtP0_1, rtPmin_k=1)
...
(rtP0_5, rtPmin_k=5)

where rtP0_X is P0 run time and rtP0_X is P0 run time again but possibly
different values as they are different runs.

2015-11-23 14:15 GMT+01:00 pierreganty notifications@github.com:

**Comment by elenagutiv** [comment]
I have just updated the python script to not measure P0 runtime for each k. Now elp-scatterplot-P0-Pk.html represents what we want.

Notice that, for some programs P0 only points (rtP0, rtPmin_k=1)(rtP0, rtPmin_k=2) (for example) are represented. This means that for k values greater than 2, ELP reached timeout and the corresponding P1[k] and P2[k], with k>2 were discarded (to me it hadn't sense to store a JSON object with P0 and P1[k] runtime but no P2[k] runtime, so I discarded the whole triple (P0,P1[k],P2[k]).
